### DISCUSSION TITLE
Retrieve user's mysite URL

### DISCUSSION THREAD

**Discussion by aolg-gith** [abraunegg/onedrive#2994] [General]
Hello,
I've run a --sync this morning after a few days. I got the message about a version of cURL/libcurl with bugs (went away smoothly using the script here https://github.com/abraunegg/onedrive/issues/2991), and the following error message as well
```
ERROR: Microsoft OneDrive API returned an error with the following message:
  Error Message:    HTTP request returned status code 404 (Not Found)
  Error Reason:     Unable to retrieve user's mysite URL.
  Error Code:       UnknownError
  Error Timestamp:  2024-11-22T10:37:16
  API Request ID:   acad37f6-2ac1-4495-aaee-4fa969047324

Unable to query OneDrive API to obtain required account details
```
I get the same with --sync, --monitor, etc... I tried to logout and re-login, but nothing changed. Do you know what is going on? Thanks :)

**Comment by abraunegg** [comment]
@aolg-gith 
> Do you know what is going on?

You appear to have a problem with your Microsoft Account.

Please raise a fault|query with Microsoft as there is zero I can do about this problem - sorry.

**References:**
* https://www.google.com/search?q=onedrive+Unable+to+retrieve+user%27s+mysite+URL&oq=onedrive+Unable+to+retrieve+user%27s+mysite+URL&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRigAdIBCDIxOTFqMWo3qAIIsAIB&sourceid=chrome&ie=UTF-8

**Reply by aolg-gith** [reply]
Ok I see, thank you!
